src/xcode/ENA/ENA/Source/Extensions/URLSessionConfiguration+Default.swift

```diff
@@ -28,4 +31,15 @@ extension URLSessionConfiguration {
 		config.requestCachePolicy = .reloadIgnoringLocalAndRemoteCacheData // avoid stale data
 		return config
 	}
+
+
+	/// Session config with caching policy `.useProtocolCachePolicy`
+	///
+	/// - Note: Caching is not realized via `NSURLCache` but via the internal SecureStore
```

Comment:
I'm neugierig why couldn't we use the default caching? Could you extend the comment here?

Reply:
Everybody was scared of potential data privacy issues so I decided to move it to our home grown store 🤷‍♂️

Reply:
Ok, I understand. Suggestion for the comment: "Because of privacy reasons, caching is not realized via `NSURLCache` but via the internal encrypted SecureStore"

Reply:
Ok, done.